src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Models/Commands.RecoveryServices.Backup.Models.csproj

```diff
@@ -44,6 +44,7 @@
   <ItemGroup>
     <Compile Include="AzureVmModels\AzureRmRecoveryServicesAzureVmJob.cs" />
     <Compile Include="AzureVmModels\AzureRmRecoveryServicesAzureVmRecoveryPoint.cs" />
+    <Compile Include="BackupEngineModels\AzureRmRecoveryServicesBackupEngine.cs" />
```

Comment:
A more appropriate name for folder might be DpmModels. (instead of BackupEngineModels)